torch/csrc/jit/serialization/pickler.h

```diff
@@ -140,7 +139,6 @@ class TORCH_API Pickler {
         memoized_class_types_(memoized_class_types),
         get_tensor_id_(std::move(get_tensor_id)),
         tag_aggregates_(tag_aggregates) {}
-  // NOLINTNEXTLINE(bugprone-exception-escape)
```

Comment:
We got rid of the exception escape?

Reply:
No, because the suppression is also given in the implementation, so it can be removed from the declaration.